features/cellular/framework/targets/UBLOX/AT/UBLOX_AT.cpp

```diff
@@ -56,6 +60,8 @@ static const intptr_t cellular_properties[AT_CellularBase::PROPERTY_MAX] = {
     0,  // PROPERTY_NON_IP_PDP_TYPE
     1,  // PROPERTY_AT_CGEREP
 };
+#else
+#error This modem is currently not supported.
```

Comment:
If you look at Travis failures, this line is breaking. 

```
[Error] UBLOX_AT.cpp@64,2: #error This modem is currently not supported.
[Error] UBLOX_AT.cpp@125,2: #error This modem is currently not supported.
[Error] UBLOX_AT.cpp@69,46: 'cellular_properties' was not declared in this scope
[Warning] UBLOX_AT.cpp@105,19: unused variable 'err' [-Wunused-variable]
```

Reply:
I will double check as build was successful for me for U201, R410 and R412. Please correct me if i am wrong but i think this happened as CI was compiling these files for target UBLOX_C030_R41XM. And since there are no modem macros set for this target, the #error would cause a failure.

Reply:
Yes, this was for unrelated target. as it compiles the code it hits this error.

Reply:
@0xc0170 I have added fix for build failures.